Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -249,7 +249,6 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 
 				e.NewElement.PropertyChanged += OnElementPropertyChanged;
 
-				SkipNextInvalidate();
```

Comment:
This code was added about 6 years ago to the slow renderer to improve performance on the ListView

At that point LabelRenderer also had a lot fewer UpdateXXXX calls so I think currently the skip is probably pointless and we'd either need to sprinkle a bunch more around or figure out a different way to block the invalidate like changing the visibility. 

This might even be an over optimization with CV